app/components/option_item/index.tsx

```diff
@@ -79,12 +93,34 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
 
 const OptionItem = ({
     action, description, destructive, icon,
-    info, label, selected,
+    info, inline = false, label, selected,
     testID = 'optionItem', type, value, containerStyle,
 }: Props) => {
     const theme = useTheme();
     const styles = getStyleSheet(theme);
 
+    const useInLine = useMemo(() => {
```

Comment:
The name of the var can create confusion with a hook. But do we need to memoize it? Maybe call it isInline?

Reply:
No need to memoize at all. It is a base type (Boolean) that is probably practically the same amount of process than the overhead of useMemo (and less memory usage).